### ISSUE TITLE
Overflow tab links are not accessible

### ISSUE THREAD

**Issue by jsunsawyer** [craftcms/cms#7813]
### Description
I'm referring to the functionality introduced in https://github.com/craftcms/cms/issues/3073

I noticed the new a11y features and was exploring keyboard navigation.

### Steps to reproduce

1. Tab to overflow tabs `...`
2. Press space bar to interact
3. Attempt to tab (skips links)
4. Instead, attempt to use arrows (no change in focus)

### Behavior Question

When selecting a tab via keyboard navigation, the next tab (keyboard key) focus is the first tab. Should the focus instead be on the content within the selected tab? I'm not sure what the expected behavior is here, but it feels less than ideal.

### Additional info

- Craft version: 3.6.12

**Comment by brandonkelly** [comment]
Thanks for pointing that out! Fixed for the next release.

**Comment by brandonkelly** [comment]
Craft 3.6.12.1 is out now with the fix for this. Thanks again!

**Comment by smockensturm** [comment]
Hi. Would this by any chance also cause that little overflow button to not respond to touch events in a mobile layout?

**Comment by brandonkelly** [comment]
@smockensturm The tabs were redesigned to be more mobile-friendly in Craft 3.7. So this change is actually no longer relevant.
